From patchwork Tue Sep  6 19:49:01 2022
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Liam Howlett <Liam.Howlett@Oracle.com>
X-Patchwork-Id: 12968094
Return-Path: <owner-linux-mm@kvack.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from kanga.kvack.org (kanga.kvack.org [205.233.56.17])
	by smtp.lore.kernel.org (Postfix) with ESMTP id D569DC38145
	for <linux-mm@archiver.kernel.org>; Tue,  6 Sep 2022 19:50:07 +0000 (UTC)
Received: by kanga.kvack.org (Postfix)
	id 1EB4D8D0005; Tue,  6 Sep 2022 15:49:53 -0400 (EDT)
Received: by kanga.kvack.org (Postfix, from userid 40)
	id 14DB280007; Tue,  6 Sep 2022 15:49:53 -0400 (EDT)
X-Delivered-To: int-list-linux-mm@kvack.org
Received: by kanga.kvack.org (Postfix, from userid 63042)
	id E6BAA8D000E; Tue,  6 Sep 2022 15:49:52 -0400 (EDT)
X-Delivered-To: linux-mm@kvack.org
Received: from relay.hostedemail.com (smtprelay0017.hostedemail.com
 [216.40.44.17])
	by kanga.kvack.org (Postfix) with ESMTP id C82238D0005
	for <linux-mm@kvack.org>; Tue,  6 Sep 2022 15:49:52 -0400 (EDT)
Received: from smtpin06.hostedemail.com (a10.router.float.18 [10.200.18.1])
	by unirelay02.hostedemail.com (Postfix) with ESMTP id AD7DE120617
	for <linux-mm@kvack.org>; Tue,  6 Sep 2022 19:49:52 +0000 (UTC)
X-FDA: 79882701024.06.AE3FBF6
Received: from mx0a-00069f02.pphosted.com (mx0a-00069f02.pphosted.com
 [205.220.165.32])
	by imf07.hostedemail.com (Postfix) with ESMTP id 3A3274005D
	for <linux-mm@kvack.org>; Tue,  6 Sep 2022 19:49:52 +0000 (UTC)
Received: from pps.filterd (m0246627.ppops.net [127.0.0.1])
	by mx0b-00069f02.pphosted.com (8.17.1.5/8.17.1.5) with ESMTP id
 286IdAfb020044;
	Tue, 6 Sep 2022 19:49:51 GMT
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=oracle.com;
 h=from : to : cc :
 subject : date : message-id : references : in-reply-to : content-type :
 content-transfer-encoding : mime-version; s=corp-2022-7-12;
 bh=7dVZ36W2B9PDvrNTLWh30F+7mrKEEJuPG6HKtfsJpKo=;
 b=Qlrsqbog8VKbAeFYDR4gmtVYagzKXsC7jCjq7myVTIxi5k2EPO80qLmdbIvnZggFJK8e
 GIb9aQ8c5VUBCPSrrm9JtYIuymV2jvyi4whS8T4iGzc0yNpNukl3rWRNEE5Id7/xgAL6
 tEK10EtonPH/DfxUhIWOJsaROaloumDi6Xa3o1OG41zGM+ogPRazLy1V49I8moa5Evx4
 J6m3SbEr6xcyXQb1DXjjWk0LsJf/FthFm5LCZQ0u0ag/BQy8Aq/LM6/40Oeg7DZKNaYo
 gQS6pwVRzXzJStaUqPcV2wgyQX+m1EAQm3cIcatfhqlmsQydGlj/c9iMLh3DWgQJWk5G rQ==
Received: from phxpaimrmta01.imrmtpd1.prodappphxaev1.oraclevcn.com
 (phxpaimrmta01.appoci.oracle.com [138.1.114.2])
	by mx0b-00069f02.pphosted.com (PPS) with ESMTPS id 3jbwh1exns-1
	(version=TLSv1.2 cipher=ECDHE-RSA-AES256-GCM-SHA384 bits=256 verify=OK);
	Tue, 06 Sep 2022 19:49:51 +0000
Received: from pps.filterd
 (phxpaimrmta01.imrmtpd1.prodappphxaev1.oraclevcn.com [127.0.0.1])
	by phxpaimrmta01.imrmtpd1.prodappphxaev1.oraclevcn.com (8.17.1.5/8.17.1.5)
 with ESMTP id 286HF67U031251;
	Tue, 6 Sep 2022 19:49:50 GMT
Received: from nam11-co1-obe.outbound.protection.outlook.com
 (mail-co1nam11lp2173.outbound.protection.outlook.com [104.47.56.173])
	by phxpaimrmta01.imrmtpd1.prodappphxaev1.oraclevcn.com (PPS) with ESMTPS id
 3jbwc3ahkd-7
	(version=TLSv1.2 cipher=ECDHE-RSA-AES256-GCM-SHA384 bits=256 verify=OK);
	Tue, 06 Sep 2022 19:49:50 +0000
ARC-Seal: i=1; a=rsa-sha256; s=arcselector9901; d=microsoft.com; cv=none;
 b=Htm6gv3p8na8C2eJtp74wUfY3bft/WkIgKYXX1k0I26SbRSe1OFyWcJNXAY/M/qDauqwWczBpysogx21VBRqieRZp1FEXAIocYosFiw+BBdFMWrMTncl4Qpu9+IfJwC8tX/Onk1hlh27PL9Vw+0BfIzbK5LmWpIur6EMv97fyRA4OXmUJMQ1nz//Y8/gQTwn22rC6YuGYMj2sljs/RsdkkFnGiMJJoC2rvrFTC7XGD3GUPca2PVQyyr22H+Xsl/9H30k2bYC7iLLik3DXDntGxniorfEA1pm/+f6ijmCxXdIR3NQVoMyJHJNxifHDbiO9/o8DFUgN60hSq5tVtMbzg==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector9901;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-AntiSpam-MessageData-ChunkCount:X-MS-Exchange-AntiSpam-MessageData-0:X-MS-Exchange-AntiSpam-MessageData-1;
 bh=7dVZ36W2B9PDvrNTLWh30F+7mrKEEJuPG6HKtfsJpKo=;
 b=md6yXl4DnqFyPptInROWtOjHJdTPjgZKwuyVqDMIYa0Mb3rVagEOpGal8Xy9SMjJOT1OR09Hhe5omTNnTwbMY3VWkciKkaUwh4wPTVXyGqY/9vJj/qJC7X6kH+MDH21ypW9wEuBDrWgPz1NTt72t1WtD46Ok5YH6/Gn2chll0SFC8qyI+BGDCv0RpRo5DLbpVlDisDUt526MCXRxz+LKtlVQHpkd7WjFHVK4VXPs+xsV1L37omf7tHNPY3KQHJwjGRXi14OoHo6Bc94jv60YF7luadLjW8IJAHai5LyjnPGX+iRsZnm5RmQpeSDCC3fCSAG2QsbZILadvLW9fcJs6Q==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=oracle.com; dmarc=pass action=none header.from=oracle.com;
 dkim=pass header.d=oracle.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=oracle.onmicrosoft.com; s=selector2-oracle-onmicrosoft-com;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=7dVZ36W2B9PDvrNTLWh30F+7mrKEEJuPG6HKtfsJpKo=;
 b=KmswoKE5YGgt49GG2eWuETScPkAEEDGxkv0AZcXKoHcdU2xfljiQ+XomrRttJWFDmP7FnBbuoaOBrjl6RE4Wvg6g+DPSmV4skWNNbxkdEQkUuNbh7D1JFp9Y2rsMDqFNJlTmCpn7m4AElCoOuXLgs1QcEQFrAiLKmoek+gJgzJA=
Received: from SN6PR10MB3022.namprd10.prod.outlook.com (2603:10b6:805:d8::25)
 by CH0PR10MB4876.namprd10.prod.outlook.com (2603:10b6:610:c9::16) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.5588.15; Tue, 6 Sep
 2022 19:49:47 +0000
Received: from SN6PR10MB3022.namprd10.prod.outlook.com
 ([fe80::a420:3107:436d:d223]) by SN6PR10MB3022.namprd10.prod.outlook.com
 ([fe80::a420:3107:436d:d223%5]) with mapi id 15.20.5588.018; Tue, 6 Sep 2022
 19:49:47 +0000
From: Liam Howlett <liam.howlett@oracle.com>
To: "maple-tree@lists.infradead.org" <maple-tree@lists.infradead.org>,
        "linux-mm@kvack.org" <linux-mm@kvack.org>,
        "linux-kernel@vger.kernel.org"
	<linux-kernel@vger.kernel.org>,
        Andrew Morton <akpm@linux-foundation.org>
CC: Liam Howlett <liam.howlett@oracle.com>
Subject: [PATCH v14 55/70] mm/memcontrol: stop using mm->highest_vm_end
Thread-Topic: [PATCH v14 55/70] mm/memcontrol: stop using mm->highest_vm_end
Thread-Index: AQHYwim2cMVZ1COXTUOVia1g90FxOw==
Date: Tue, 6 Sep 2022 19:49:01 +0000
Message-ID: <20220906194824.2110408-56-Liam.Howlett@oracle.com>
References: <20220906194824.2110408-1-Liam.Howlett@oracle.com>
In-Reply-To: <20220906194824.2110408-1-Liam.Howlett@oracle.com>
Accept-Language: en-US
Content-Language: en-US
X-MS-Has-Attach: 
X-MS-TNEF-Correlator: 
x-mailer: git-send-email 2.35.1
x-ms-publictraffictype: Email
x-ms-office365-filtering-correlation-id: e504a16f-73b9-4c95-5313-08da9040f465
x-ms-traffictypediagnostic: CH0PR10MB4876:EE_
x-ms-exchange-senderadcheck: 1
x-ms-exchange-antispam-relay: 0
x-microsoft-antispam: BCL:0;
x-microsoft-antispam-message-info: 
 1dGuaU8GIR7HhE3EXwb9+UYuzinj5vwWUbFOeHwgFQt/g2DqPSN1PLWhK0hFTJElBPv8j8dhWhBjq8SKb5V72ub7bg78nwfQWeLtWnJ+z3afkaQQZ6gMkQV6pqDnb5iVgfQi+LEQ7o6lADwvEhZxrRn/vrnELA3/X4NngschNIM6Hls5Ztf3XYfyTaVIWS7kqgP81RjM5Q9A1Lx2G0aSO/6oQaPbZMLP2zT47W70JzstWI7zDD8JpeNfRSd4OLlBl5m+rXK2Bd++x5kHhJieKj8dwivY1nbqV4j3miIKK9BfMWs2ki+WR2wBnjldHAAExIrSJEq+XMAw02PCMAlxFnqx/rUbhYsz4fb1mGXEvYkygfmXk5ijv2kqpANLL56dUCcxWJbWKqbH9M9psQKZOU5Gfwc9S0VonshjGHlH0NNVsh8Zfd8kZ2nnlNCW81xzQooUfyXF0CuX4nr+siW/H6q+fVb6LEg3blIb9czT9oq4cuao6SYKPUPdDhYljdvI/pAk36ne9j+vHITyuV0JJJenqhJM+apS7ewJNNmC3LiZgSR+kvaxuOyjT2pkSxHE9lZ5DC1xHNkgxKEqxOZal+gZPZCc+A+kwQV8xnin3OnITg2OZlxHliZTDrxnqCasj8y/ZAod2LYg1uIF93gsbJN3Sf3tCcgouywKVjwwzZo86L0iACBZ0J7/jFJ1OkoZc1AMdiYXBQnZhR6dLtCGAH+vT6hbG75+izdnf5sJKg1Qbgv+0tSxCjHfClCoUt8CGZdNATtweN+KebiXy/BPBQ==
x-forefront-antispam-report: 
 CIP:255.255.255.255;CTRY:;LANG:en;SCL:1;SRV:;IPV:NLI;SFV:NSPM;H:SN6PR10MB3022.namprd10.prod.outlook.com;PTR:;CAT:NONE;SFS:(13230016)(39860400002)(136003)(366004)(376002)(396003)(346002)(8936002)(6486002)(8676002)(4326008)(91956017)(122000001)(66946007)(66446008)(186003)(66476007)(76116006)(64756008)(66556008)(2616005)(38100700002)(38070700005)(478600001)(83380400001)(36756003)(1076003)(5660300002)(41300700001)(71200400001)(6666004)(44832011)(26005)(6506007)(316002)(86362001)(107886003)(2906002)(110136005)(6512007);DIR:OUT;SFP:1101;
x-ms-exchange-antispam-messagedata-chunkcount: 1
x-ms-exchange-antispam-messagedata-0: =?iso-8859-1?q?4DyU9frA4w8whxCu34gj9pv?=
	=?iso-8859-1?q?w5jP79sNJYgSxrftH7SPxo+dIMqIu/VpJK1zQ5Dv0W46I6Mssj8OBjUruGy8?=
	=?iso-8859-1?q?UaIS1DGXIQuM1AxCPVXZAhC0fkzrcgtvTME8uxvl3F2+4TYqgZKht0twMfmU?=
	=?iso-8859-1?q?0L8vlIkCaOcFCOEJli6ChLR4abqpuHap7O7isaXlweidSTNE3AN/n9r8Zsnf?=
	=?iso-8859-1?q?6C4Qj1UookcirKZJve5EeGBFbLygcNkOMfi1Q4pmUjX5a4lE9AZDT48Pzo0x?=
	=?iso-8859-1?q?krSm10ElZJm/cTZLCduSOs6tOMz7lj7EdlQLdZHl/9t4pAQ70VL/hCj9SzWX?=
	=?iso-8859-1?q?cAk+qeDil/yGlYXbBqJwojo6i8B8AHRtAv3TNrdQr/wuseYaqMOFQO3zGmlM?=
	=?iso-8859-1?q?sAj/P5PUSWVEj7mDdJuHZt30u86YXIwW5zWlmngxPl1FExPKdbKgpupjBgTC?=
	=?iso-8859-1?q?6y0luTRK8A0+d8AX+VlfpcWNPFdEO9gLu3I6HMSeY9gmPluUKA5u+iGdtOUt?=
	=?iso-8859-1?q?62lEw1noLlT1Tgo7kOJa7nK/qR9/nFeSYEFKLfc+FbgsJIX2/y5BAl2NvFyu?=
	=?iso-8859-1?q?cxxSxYnd3EM9mogq9Hi1+x+UvEw2l4dnBXCl+cU3I61R8W/KEgr4hJQhZkhE?=
	=?iso-8859-1?q?YS+yrqskyWfMyqouOPzAd9lohwHwOprWwKCRIPnAwIE7CYl/2/6BfaCVjJwB?=
	=?iso-8859-1?q?p0bcK/TX3HNL6QUNO3DSFvb5iFl8jgp35cF5H8YGA1fNE+5bSPNytdaFLMlj?=
	=?iso-8859-1?q?YNGktSORi5k3SmZJcSxFPEuTi2cd9vXMbNt/jswpEK3EDNHrmrp53SicJ+pV?=
	=?iso-8859-1?q?0JnZ1FJQ2zEzhzh5q0yJEOu5jEv2GEIUy3Sp9q/vXTTSWSZh4ODrhPykx11Z?=
	=?iso-8859-1?q?ADbbpfXJRHEsqRRyLxZmnyEYBbMlxq33AESoAYDvwbAZyq1LKFpmWA/ysOB8?=
	=?iso-8859-1?q?4TuFIV1ujDV+rD9IiikNqnXuHa7+O1Dipq1l+WqnQQJRWsznk7o/7T/V8ysL?=
	=?iso-8859-1?q?3DUYo/mnuTWXOPQVsX1QsNU0g0y3M89TOCUcXxulmmr60vX9dRGOHYJv5ZWn?=
	=?iso-8859-1?q?tF1JtNc0CkR7VWmW7XBODOI7zzw/z6l8HJ39lAC/OFi0Jn/MtIouDJCw1uQN?=
	=?iso-8859-1?q?IECmURj5sLxGHM2j0ucu7HzaYlxbb7jkrv0pFcN5FfaEkt0crfChSrOcBI7W?=
	=?iso-8859-1?q?DK3wwbee+bih/T2vbi3v/LRyOV0zZVeaCQRTU+ErHmKAzhLlmhBoEIXDttQm?=
	=?iso-8859-1?q?XFm3g5LJRVUw6dsyaUPL77g76sqkO8jCqcLIRk4bSRmSkBoLBVoJVS+3vlCz?=
	=?iso-8859-1?q?/CTerw7XEg7DN7AZ6S/n1pcgc+2KXhj8v+iC7Dh1NJkXmRNEQ0MjpjFoUceM?=
	=?iso-8859-1?q?2UKJpQx1xPjJIJnd8RzRSVho+bSzw/EcfDaMtzxCEJ1HrLF7adqXYfam+gzs?=
	=?iso-8859-1?q?ydZ3T0MzKOPOmvrBOMjKp7iHu/vDPAff+6zekV5dT//6/xaPEsFmDioGtB/Z?=
	=?iso-8859-1?q?fjnhOC9pU75mqgIonMviYxVVnrOINEKKPQjBAtrf5nLsDTkPdlZa7b3WrEqh?=
	=?iso-8859-1?q?MWCV4e7uTwnE9N0eZcxufuhG6V18VUSX2WhJuBeYs6CZjkCEoXsboh5E4W82?=
	=?iso-8859-1?q?b+5JyvPj3q0s6Y7hcRBZlLnBLL/Zi7Tw78f6lnA=3D=3D?=
MIME-Version: 1.0
X-OriginatorOrg: oracle.com
X-MS-Exchange-CrossTenant-AuthAs: Internal
X-MS-Exchange-CrossTenant-AuthSource: SN6PR10MB3022.namprd10.prod.outlook.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 e504a16f-73b9-4c95-5313-08da9040f465
X-MS-Exchange-CrossTenant-originalarrivaltime: 06 Sep 2022 19:49:01.7715
 (UTC)
X-MS-Exchange-CrossTenant-fromentityheader: Hosted
X-MS-Exchange-CrossTenant-id: 4e2c6054-71cb-48f1-bd6c-3a9705aca71b
X-MS-Exchange-CrossTenant-mailboxtype: HOSTED
X-MS-Exchange-CrossTenant-userprincipalname: 
 NN61pm5J0XxoBKdqnwtRliV3PeBWSqJcuyIa/N8nbJCq/AKvLi7UlhQguVwOC/jZbBpBFhl3iDZsD4ghD5VaoQ==
X-MS-Exchange-Transport-CrossTenantHeadersStamped: CH0PR10MB4876
X-Proofpoint-Virus-Version: vendor=baseguard
 engine=ICAP:2.0.205,Aquarius:18.0.895,Hydra:6.0.528,FMLib:17.11.122.1
 definitions=2022-09-06_09,2022-09-06_02,2022-06-22_01
X-Proofpoint-Spam-Details: rule=notspam policy=default score=0 spamscore=0
 malwarescore=0 mlxscore=0
 suspectscore=0 mlxlogscore=999 phishscore=0 adultscore=0 bulkscore=0
 classifier=spam adjust=0 reason=mlx scancount=1 engine=8.12.0-2207270000
 definitions=main-2209060091
X-Proofpoint-GUID: W8fPS8yXG9A7spzon2-8lTJs1vifEGdh
X-Proofpoint-ORIG-GUID: W8fPS8yXG9A7spzon2-8lTJs1vifEGdh
ARC-Seal: i=2; s=arc-20220608; d=hostedemail.com; t=1662493792; a=rsa-sha256;
	cv=pass;
	b=A5V3Ac0zRf//2KBvcZaab+8Vk4sF7TKPIXAo6ucgpqDhgaFQQrfXFrek0PuEx/YOd6jTBk
	B6drSiSz6gtoxL5pJxP5YdBN+XXggnKDVDri3zX85EO1bGhQHcX81QMhNxqnpVfPWKKeD2
	n2KtrTKoMfsvNs0/mgnDhYMG5jNyEoU=
ARC-Authentication-Results: i=2;
	imf07.hostedemail.com;
	dkim=pass header.d=oracle.com header.s=corp-2022-7-12 header.b=Qlrsqbog;
	dkim=pass header.d=oracle.onmicrosoft.com
 header.s=selector2-oracle-onmicrosoft-com header.b=KmswoKE5;
	spf=pass (imf07.hostedemail.com: domain of liam.howlett@oracle.com designates
 205.220.165.32 as permitted sender) smtp.mailfrom=liam.howlett@oracle.com;
	dmarc=pass (policy=none) header.from=oracle.com;
	arc=pass ("microsoft.com:s=arcselector9901:i=1")
ARC-Message-Signature: i=2; a=rsa-sha256; c=relaxed/relaxed;
 d=hostedemail.com;
	s=arc-20220608; t=1662493792;
	h=from:from:sender:reply-to:subject:subject:date:date:
	 message-id:message-id:to:to:cc:cc:mime-version:mime-version:
	 content-type:content-type:
	 content-transfer-encoding:content-transfer-encoding:
	 in-reply-to:in-reply-to:references:references:dkim-signature;
	bh=7dVZ36W2B9PDvrNTLWh30F+7mrKEEJuPG6HKtfsJpKo=;
	b=IefZWkzPFHVm4oQ5X1f89zJL3wbMdNWkzwDdjlLXqkpPHuxlrJtAd2mA0R7YD7hJm5iAZR
	gs05XqkRAuHGKF237XEDJ5JWQEbkw4xeizbyO8yHrRnR1vdbhLaGM9dHJuli/gyGzrd5MD
	n1QkYJIw/GTaSm42LU8GmFgIHQske6E=
Authentication-Results: imf07.hostedemail.com;
	dkim=pass header.d=oracle.com header.s=corp-2022-7-12 header.b=Qlrsqbog;
	dkim=pass header.d=oracle.onmicrosoft.com
 header.s=selector2-oracle-onmicrosoft-com header.b=KmswoKE5;
	spf=pass (imf07.hostedemail.com: domain of liam.howlett@oracle.com designates
 205.220.165.32 as permitted sender) smtp.mailfrom=liam.howlett@oracle.com;
	dmarc=pass (policy=none) header.from=oracle.com;
	arc=pass ("microsoft.com:s=arcselector9901:i=1")
X-Rspam-User: 
X-Rspamd-Server: rspam06
X-Stat-Signature: mmr4ga55d3jwkz487qcagqxczwkcb3ku
X-Rspamd-Queue-Id: 3A3274005D
X-HE-Tag: 1662493792-740364
X-Bogosity: Ham, tests=bogofilter, spamicity=0.000000, version=1.2.4
Sender: owner-linux-mm@kvack.org
Precedence: bulk
X-Loop: owner-majordomo@kvack.org
List-ID: <linux-mm.kvack.org>

From: "Liam R. Howlett" <Liam.Howlett@Oracle.com>

Pass through ULONG_MAX instead.

Signed-off-by: Liam R. Howlett <Liam.Howlett@Oracle.com>
---
 mm/memcontrol.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/mm/memcontrol.c b/mm/memcontrol.c
index 5e0fa278eb53..668b6792fd1c 100644
--- a/mm/memcontrol.c
+++ b/mm/memcontrol.c
@@ -5879,7 +5879,7 @@ static unsigned long mem_cgroup_count_precharge(struct mm_struct *mm)
 	unsigned long precharge;
 
 	mmap_read_lock(mm);
-	walk_page_range(mm, 0, mm->highest_vm_end, &precharge_walk_ops, NULL);
+	walk_page_range(mm, 0, ULONG_MAX, &precharge_walk_ops, NULL);
 	mmap_read_unlock(mm);
 
 	precharge = mc.precharge;
@@ -6177,9 +6177,7 @@ static void mem_cgroup_move_charge(void)
 	 * When we have consumed all precharges and failed in doing
 	 * additional charge, the page walk just aborts.
 	 */
-	walk_page_range(mc.mm, 0, mc.mm->highest_vm_end, &charge_walk_ops,
-			NULL);
-
+	walk_page_range(mc.mm, 0, ULONG_MAX, &charge_walk_ops, NULL);
 	mmap_read_unlock(mc.mm);
 	atomic_dec(&mc.from->moving_account);
 }

